hyp3_metadata/rtc.py

```diff
@@ -113,3 +118,54 @@ def create_metadata_file(cls, payload: dict, template: str, reference_file: Path
             f.write(content)
 
         return output_file
+
+
+def get_dem_template_id(dem_name: str) -> Optional[str]:
+    if dem_name.startswith('EU'):
+        return 'eu'
+    if dem_name.startswith('GIMP'):
+        return 'gimp'
+    if dem_name.startswith('IFSAR'):
+        return 'ifsar'
+    if dem_name.startswith('NED'):
+        return 'ned'
+    if dem_name.startswith('REMA'):
+        return 'rema'
+    if dem_name.startswith('SRTM'):
+        return 'srtm'
+    if dem_name == 'GLO-30':
+        return 'cop'
```

Comment:
This feels more like a `util` function even though RTC is the only thing leveraging different DEMs